README.md

```diff
@@ -58,16 +58,6 @@ view3D.updateLuts(aimg);
 ```
 
 # React example 
-- in `webpack.config.js`
-```JavaScript
-    rules: [
-        ...,
-        {
-            test: /Worker\.js$/,
-            use: 'worker-loader?inline=true'
-        }
-    ]
-```
 - in `VolumeViewer.jsx`
 ```JavaScript 
```

Comment:
I know this is not part of this PR but I just noticed that this code block is not closed properly. The backticks in line 131 need to be un-indented. 

![image](https://user-images.githubusercontent.com/12690133/128555302-b8b84789-dc9c-4458-bc0a-015fdacff727.png)

Reply:
Thanks! will fix that